File: autograder/testcase_types/cpython/templates/testcase_template1.py
# student_submission is provided by autograder, represents student's module, and can be used freely
# PASS(), RESULT(res), FAIL(), and CHECK_STDOUT() are also provided by autograder and can be used freely


def main():
    # You can call any function from student's file like this.
    # It can have any arguments and return values -- use it like any other function.
    result = student_submission.some_student_function()

    SOME_EXPECTED_RESULT = 83
    if result == SOME_EXPECTED_RESULT:
        PASS()
    else:
        FAIL()


main()

File: autograder/testcase_types/gcc/c++_templates/testcase_template1.cpp
// All non-static functions in student's file are provided by autograder and can be used freely
// PASS(), RESULT(res), FAIL(), and CHECK_STDOUT() are also provided by autograder and can be used freely

// If you want to call a function from student's file in C/C++, you have to prototype it in the same way you'd prototype any of your functions.
int some_student_function();

int main()
{
    // You can call any non-static function from student's file like this.
    // It can have any arguments and return values -- use it like any other function.
    int result = some_student_function();

    int SOME_EXPECTED_RESULT = 83;
    if (result == SOME_EXPECTED_RESULT)
        PASS();
    else
        FAIL();
}